app/assets/javascripts/admin/all.js

```diff
@@ -19,7 +19,10 @@
 //= require angular
 //= require angular-resource
 //= require angular-animate
-//= require angular-sanitize
+// angular-sanitize is replaced by textAngular-sanitize further down.
+// We should include only one.
+// https://github.com/textAngular/textAngular#where-to-get-it
+// require angular-sanitize
```

Comment:
This is removing the dependency so I think you should remove all these lines here including the comment and just add a simple line comment above the textangular-sanitize require:

```
// this replaces angular-sanitize
//= require textAngular-sanitize.min.js

```

Reply:
the reference to "We should include only one." and link would go to the commit message.

Reply:
Good idea. I added a commit with that. :heavy_check_mark: